ruby/ql/lib/codeql/files/FilesImpl.qll

```diff
@@ -0,0 +1,21 @@
+private import codeql.utils.FileSystem as FS
+
+module FilesImpl implements FS::FilesSig {
+  class AtFile extends @file, AtContainer {
+    override string getAbsolutePath() { files(this, result) }
+  }
+
+  abstract class AtContainer extends @container {
```

Comment:
Let's remove the `At` prefix, they were there to indicate that those types mapped to an `@` type.